### ISSUE TITLE
Crashes when playing a shuffled music playlist (new music backend)

### ISSUE THREAD

**Issue by BobbyWibowo** [jellyfin/jellyfin-androidtv#3248]
### Describe the bug

1. Have a music playlist
2. Play the playlist with "Shuffle all" button
3. Let some tracks play through
4. ...?

I'm not sure if this issue is specific to shuffled playlist or if it happens to all playlists in general, but I indeed used to always just shuffle them.

Do note that I did not disable the new music backend (the option was not ticked), but I'm currently testing playlist playback with it turned on now. Will update if there's any new findings.

**UPDATE:** A few hours later, can confirm this only happens with the new music backend.

### Logs

***Stack Trace***: 
```log
java.lang.IllegalArgumentException: currentMediaItemIndex must be less than playlist.size()
	at androidx.media3.common.util.Assertions.checkArgument(Assertions.java:55)
	at androidx.media3.common.SimpleBasePlayer$State.<init>(SimpleBasePlayer.java:933)
	at androidx.media3.common.SimpleBasePlayer$State.<init>(SimpleBasePlayer.java:100)
	at androidx.media3.common.SimpleBasePlayer$State$Builder.build(SimpleBasePlayer.java:769)
	at org.jellyfin.playback.core.mediasession.MediaSessionPlayer.getState(MediaSessionPlayer.kt:132)
	at androidx.media3.common.SimpleBasePlayer.invalidateState(SimpleBasePlayer.java:2920)
	at org.jellyfin.playback.core.mediasession.MediaSessionPlayer.access$invalidateState(MediaSessionPlayer.kt:29)
	at org.jellyfin.playback.core.mediasession.MediaSessionPlayer$invalidateStateOnEach$1$1.invokeSuspend(MediaSessionPlayer.kt:49)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:201)
	at android.os.Looper.loop(Looper.java:288)
	at android.app.ActivityThread.main(ActivityThread.java:7839)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003)
	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@ccc7bba, Dispatchers.Default]

```  
***Logcat***: 
```log
12-24 10:57:12.979 I/DreamHostKt( 1612): Loading random library showcase item baf001fd-a079-673c-76b0-82e87d0760fa
12-24 10:57:17.095 D/BufferPoolAccessor2.0( 1612): bufferpool2 0xc2a7c958 : 5(163840 size) total buffers - 1(32768 size) used buffers - 952/957 (recycle/alloc) - 5/1910 (fetch/transfer)
12-24 10:57:22.473 D/BufferPoolAccessor2.0( 1612): bufferpool2 0xc2a7c958 : 5(163840 size) total buffers - 1(32768 size) used buffers - 1008/1013 (recycle/alloc) - 5/2022 (fetch/transfer)
12-24 10:57:27.851 D/BufferPoolAccessor2.0( 1612): bufferpool2 0xc2a7c958 : 5(163840 size) total buffers - 1(32768 size) used buffers - 1064/1069 (recycle/alloc) - 5/2134 (fetch/transfer)
12-24 10:57:33.223 D/BufferPoolAccessor2.0( 1612): bufferpool2 0xc2a7c958 : 5(163840 size) total buffers - 1(32768 size) used buffers - 1120/1125 (recycle/alloc) - 5/2246 (fetch/transfer)
12-24 10:57:38.603 D/BufferPoolAccessor2.0( 1612): bufferpool2 0xc2a7c958 : 5(163840 size) total buffers - 1(32768 size) used buffers - 1176/1181 (recycle/alloc) - 5/2358 (fetch/transfer)
12-24 10:57:39.490 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession( 1612): Sending (raw) message {"MessageType":"KeepAlive","Data":{}}
12-24 10:57:39.497 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession( 1612): Receiving (raw) message {"MessageType":"KeepAlive","MessageId":"50a71dfa16504948b81064588bba39e1"}
12-24 10:57:43.978 D/BufferPoolAccessor2.0( 1612): bufferpool2 0xc2a7c958 : 5(163840 size) total buffers - 1(32768 size) used buffers - 1232/1237 (recycle/alloc) - 5/2470 (fetch/transfer)
12-24 10:57:45.090 I/org.jellyfin.sdk.api.client.KtorClient( 1612): GET http://192.168.0.100:8096/Users/bb7ea459-51a4-4e6e-a639-6507a5fb329f/Items?limit=5&recursive=true&includeItemTypes=Movie&includeItemTypes=Series&imageTypes=Backdrop&sortBy=Random&enableTotalRecordCount=true&enableImages=true
12-24 10:57:45.140 I/DreamHostKt( 1612): Loading random library showcase item 81e256dc-9f16-ba44-94a1-7a7ea98e9f96
12-24 10:57:49.353 D/BufferPoolAccessor2.0( 1612): bufferpool2 0xc2a7c958 : 5(163840 size) total buffers - 1(32768 size) used buffers - 1288/1293 (recycle/alloc) - 5/2582 (fetch/transfer)
12-24 10:57:54.727 D/BufferPoolAccessor2.0( 1612): bufferpool2 0xc2a7c958 : 5(163840 size) total buffers - 1(32768 size) used buffers - 1344/1349 (recycle/alloc) - 5/2694 (fetch/transfer)
12-24 10:58:00.098 D/BufferPoolAccessor2.0( 1612): bufferpool2 0xc2a7c958 : 5(163840 size) total buffers - 1(32768 size) used buffers - 1400/1405 (recycle/alloc) - 5/2806 (fetch/transfer)
12-24 10:58:05.475 D/BufferPoolAccessor2.0( 1612): bufferpool2 0xc2a7c958 : 5(163840 size) total buffers - 1(32768 size) used buffers - 1456/1461 (recycle/alloc) - 5/2918 (fetch/transfer)
12-24 10:58:09.490 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession( 1612): Sending (raw) message {"MessageType":"KeepAlive","Data":{}}
12-24 10:58:09.494 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession( 1612): Receiving (raw) message {"MessageType":"KeepAlive","MessageId":"1dac73f18a6643ab990447a64dfbf628"}
12-24 10:58:10.850 D/BufferPoolAccessor2.0( 1612): bufferpool2 0xc2a7c958 : 5(163840 size) total buffers - 1(32768 size) used buffers - 1512/1517 (recycle/alloc) - 5/3030 (fetch/transfer)
12-24 10:58:15.618 I/org.jellyfin.sdk.api.client.KtorClient( 1612): GET http://192.168.0.100:8096/Users/bb7ea459-51a4-4e6e-a639-6507a5fb329f/Items?limit=5&recursive=true&includeItemTypes=Movie&includeItemTypes=Series&imageTypes=Backdrop&sortBy=Random&enableTotalRecordCount=true&enableImages=true
12-24 10:58:15.656 I/DreamHostKt( 1612): Loading random library showcase item be8b1862-27e0-15d4-8dc1-6a9e525c2de6
12-24 10:58:16.226 D/BufferPoolAccessor2.0( 1612): bufferpool2 0xc2a7c958 : 5(163840 size) total buffers - 1(32768 size) used buffers - 1568/1573 (recycle/alloc) - 5/3142 (fetch/transfer)
12-24 10:58:16.982 I/lyfin.androidt( 1612): Background young concurrent copying GC freed 371138(11MB) AllocSpace objects, 1(20KB) LOS objects, 16% free, 46MB/55MB, paused 385us total 101.778ms
12-24 10:58:21.607 D/BufferPoolAccessor2.0( 1612): bufferpool2 0xc2a7c958 : 5(163840 size) total buffers - 1(32768 size) used buffers - 1624/1629 (recycle/alloc) - 5/3254 (fetch/transfer)
12-24 10:58:26.976 D/BufferPoolAccessor2.0( 1612): bufferpool2 0xc2a7c958 : 5(163840 size) total buffers - 1(32768 size) used buffers - 1680/1685 (recycle/alloc) - 5/3366 (fetch/transfer)
12-24 10:58:32.361 D/BufferPoolAccessor2.0( 1612): bufferpool2 0xc2a7c958 : 5(163840 size) total buffers - 1(32768 size) used buffers - 1736/1741 (recycle/alloc) - 5/3478 (fetch/transfer)
12-24 10:58:37.736 D/BufferPoolAccessor2.0( 1612): bufferpool2 0xc2a7c958 : 5(163840 size) total buffers - 1(32768 size) used buffers - 1792/1797 (recycle/alloc) - 5/3590 (fetch/transfer)
12-24 10:58:38.584 I/lyfin.androidt( 1612): Background concurrent copying GC freed 303389(23MB) AllocSpace objects, 4(80KB) LOS objects, 42% free, 32MB/56MB, paused 156us total 209.668ms
12-24 10:58:39.491 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession( 1612): Sending (raw) message {"MessageType":"KeepAlive","Data":{}}
12-24 10:58:39.495 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession( 1612): Receiving (raw) message {"MessageType":"KeepAlive","MessageId":"e6e8e9ba31c0484281341a9a5e8a0025"}
12-24 10:58:43.107 D/BufferPoolAccessor2.0( 1612): bufferpool2 0xc2a7c958 : 5(163840 size) total buffers - 1(32768 size) used buffers - 1848/1853 (recycle/alloc) - 5/3702 (fetch/transfer)
12-24 10:58:48.144 I/org.jellyfin.sdk.api.client.KtorClient( 1612): GET http://192.168.0.100:8096/Users/bb7ea459-51a4-4e6e-a639-6507a5fb329f/Items?limit=5&recursive=true&includeItemTypes=Movie&includeItemTypes=Series&imageTypes=Backdrop&sortBy=Random&enableTotalRecordCount=true&enableImages=true
12-24 10:58:48.203 I/DreamHostKt( 1612): Loading random library showcase item f27bb24d-cf08-290b-b436-34a7eb22509c
12-24 10:58:48.484 D/BufferPoolAccessor2.0( 1612): bufferpool2 0xc2a7c958 : 5(163840 size) total buffers - 1(32768 size) used buffers - 1904/1909 (recycle/alloc) - 5/3814 (fetch/transfer)
12-24 10:58:53.868 D/BufferPoolAccessor2.0( 1612): bufferpool2 0xc2a7c958 : 5(163840 size) total buffers - 1(32768 size) used buffers - 1960/1965 (recycle/alloc) - 5/3926 (fetch/transfer)
12-24 10:58:59.238 D/BufferPoolAccessor2.0( 1612): bufferpool2 0xc2a7c958 : 5(163840 size) total buffers - 1(32768 size) used buffers - 2016/2021 (recycle/alloc) - 5/4038 (fetch/transfer)
12-24 10:59:04.618 D/BufferPoolAccessor2.0( 1612): bufferpool2 0xc2a7c958 : 5(163840 size) total buffers - 1(32768 size) used buffers - 2072/2077 (recycle/alloc) - 5/4150 (fetch/transfer)
12-24 10:59:09.491 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession( 1612): Sending (raw) message {"MessageType":"KeepAlive","Data":{}}
12-24 10:59:09.496 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession( 1612): Receiving (raw) message {"MessageType":"KeepAlive","MessageId":"51323764ce9747439dec183650bd0580"}
12-24 10:59:09.993 D/BufferPoolAccessor2.0( 1612): bufferpool2 0xc2a7c958 : 5(163840 size) total buffers - 1(32768 size) used buffers - 2128/2133 (recycle/alloc) - 5/4262 (fetch/transfer)
12-24 10:59:15.367 D/BufferPoolAccessor2.0( 1612): bufferpool2 0xc2a7c958 : 5(163840 size) total buffers - 1(32768 size) used buffers - 2184/2189 (recycle/alloc) - 5/4374 (fetch/transfer)
12-24 10:59:18.493 I/org.jellyfin.sdk.api.client.KtorClient( 1612): GET http://192.168.0.100:8096/Users/bb7ea459-51a4-4e6e-a639-6507a5fb329f/Items?limit=5&recursive=true&includeItemTypes=Movie&includeItemTypes=Series&imageTypes=Backdrop&sortBy=Random&enableTotalRecordCount=true&enableImages=true
12-24 10:59:18.541 I/DreamHostKt( 1612): Loading random library showcase item 9c2db564-2191-5d0d-f153-0edc0503d300
12-24 10:59:20.746 D/BufferPoolAccessor2.0( 1612): bufferpool2 0xc2a7c958 : 5(163840 size) total buffers - 1(32768 size) used buffers - 2240/2245 (recycle/alloc) - 5/4486 (fetch/transfer)
12-24 10:59:26.122 D/BufferPoolAccessor2.0( 1612): bufferpool2 0xc2a7c958 : 5(163840 size) total buffers - 1(32768 size) used buffers - 2296/2301 (recycle/alloc) - 5/4598 (fetch/transfer)
12-24 10:59:31.492 D/BufferPoolAccessor2.0( 1612): bufferpool2 0xc2a7c958 : 5(163840 size) total buffers - 1(32768 size) used buffers - 2352/2357 (recycle/alloc) - 5/4710 (fetch/transfer)
12-24 10:59:36.866 D/BufferPoolAccessor2.0( 1612): bufferpool2 0xc2a7c958 : 5(163840 size) total buffers - 1(32768 size) used buffers - 2408/2413 (recycle/alloc) - 5/4822 (fetch/transfer)
12-24 10:59:37.546 I/lyfin.androidt( 1612): Background young concurrent copying GC freed 691064(23MB) AllocSpace objects, 4(80KB) LOS objects, 41% free, 32MB/56MB, paused 325us total 112.245ms
12-24 10:59:39.492 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession( 1612): Sending (raw) message {"MessageType":"KeepAlive","Data":{}}
12-24 10:59:39.499 I/org.jellyfin.sdk.api.sockets.OkHttpWebsocketSession( 1612): Receiving (raw) message {"MessageType":"KeepAlive","MessageId":"a318422e05d84808bd59ee6072816f61"}
12-24 10:59:42.244 D/BufferPoolAccessor2.0( 1612): bufferpool2 0xc2a7c958 : 5(163840 size) total buffers - 1(32768 size) used buffers - 2464/2469 (recycle/alloc) - 5/4934 (fetch/transfer)
12-24 10:59:47.619 D/BufferPoolAccessor2.0( 1612): bufferpool2 0xc2a7c958 : 5(163840 size) total buffers - 1(32768 size) used buffers - 2520/2525 (recycle/alloc) - 5/5046 (fetch/transfer)
12-24 10:59:48.796 I/org.jellyfin.sdk.api.client.KtorClient( 1612): GET http://192.168.0.100:8096/Users/bb7ea459-51a4-4e6e-a639-6507a5fb329f/Items?limit=5&recursive=true&includeItemTypes=Movie&includeItemTypes=Series&imageTypes=Backdrop&sortBy=Random&enableTotalRecordCount=true&enableImages=true
12-24 10:59:48.841 I/DreamHostKt( 1612): Loading random library showcase item be7bc15f-bff9-8d54-5d9a-8f95d186f05a
12-24 10:59:52.994 D/BufferPoolAccessor2.0( 1612): bufferpool2 0xc2a7c958 : 5(163840 size) total buffers - 1(32768 size) used buffers - 2576/2581 (recycle/alloc) - 5/5158 (fetch/transfer)
12-24 10:59:55.682 D/AudioTrack( 1612): stop(309): called with 11994240 frames delivered
12-24 10:59:56.182 D/AudioNowPlayingFragment( 1612): **** Got playstate change: IDLE
12-24 10:59:56.183 D/AudioNowPlayingFragment( 1612): Updating buttons
12-24 10:59:56.206 D/DefaultMediaStreamState( 1612): Queue entry changed to org.jellyfin.playback.jellyfin.queue.item.BaseItemDtoUserQueueEntry@ee405b9
12-24 10:59:56.207 I/org.jellyfin.sdk.api.client.KtorClient( 1612): POST http://192.168.0.100:8096/Items/63b6ee2c-f493-65ec-baff-271fcd87232e/PlaybackInfo
12-24 10:59:56.225 I/org.jellyfin.sdk.api.client.KtorClient( 1612): POST http://192.168.0.100:8096/Sessions/Playing/Stopped
12-24 10:59:56.261 W/View    ( 1612): requestLayout() improperly called by androidx.leanback.widget.HorizontalGridView{3c7c789 VFE...... .....AI. 0,0-1920,472 #7f0b026f app:id/row_content} during layout: running second layout pass
12-24 10:59:56.285 E/ACRA    ( 1612): ACRA caught a IllegalArgumentException for org.jellyfin.androidtv
12-24 10:59:56.285 E/ACRA    ( 1612): java.lang.IllegalArgumentException: currentMediaItemIndex must be less than playlist.size()
12-24 10:59:56.285 E/ACRA    ( 1612): 	at androidx.media3.common.util.Assertions.checkArgument(Assertions.java:55)
12-24 10:59:56.285 E/ACRA    ( 1612): 	at androidx.media3.common.SimpleBasePlayer$State.<init>(SimpleBasePlayer.java:933)
12-24 10:59:56.285 E/ACRA    ( 1612): 	at androidx.media3.common.SimpleBasePlayer$State.<init>(SimpleBasePlayer.java:100)
12-24 10:59:56.285 E/ACRA    ( 1612): 	at androidx.media3.common.SimpleBasePlayer$State$Builder.build(SimpleBasePlayer.java:769)
12-24 10:59:56.285 E/ACRA    ( 1612): 	at org.jellyfin.playback.core.mediasession.MediaSessionPlayer.getState(MediaSessionPlayer.kt:132)
12-24 10:59:56.285 E/ACRA    ( 1612): 	at androidx.media3.common.SimpleBasePlayer.invalidateState(SimpleBasePlayer.java:2920)
12-24 10:59:56.285 E/ACRA    ( 1612): 	at org.jellyfin.playback.core.mediasession.MediaSessionPlayer.access$invalidateState(MediaSessionPlayer.kt:29)
12-24 10:59:56.285 E/ACRA    ( 1612): 	at org.jellyfin.playback.core.mediasession.MediaSessionPlayer$invalidateStateOnEach$1$1.invokeSuspend(MediaSessionPlayer.kt:49)
12-24 10:59:56.285 E/ACRA    ( 1612): 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
12-24 10:59:56.285 E/ACRA    ( 1612): 	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
12-24 10:59:56.285 E/ACRA    ( 1612): 	at android.os.Handler.handleCallback(Handler.java:938)
12-24 10:59:56.285 E/ACRA    ( 1612): 	at android.os.Handler.dispatchMessage(Handler.java:99)
12-24 10:59:56.285 E/ACRA    ( 1612): 	at android.os.Looper.loopOnce(Looper.java:201)
12-24 10:59:56.285 E/ACRA    ( 1612): 	at android.os.Looper.loop(Looper.java:288)
12-24 10:59:56.285 E/ACRA    ( 1612): 	at android.app.ActivityThread.main(ActivityThread.java:7839)
12-24 10:59:56.285 E/ACRA    ( 1612): 	at java.lang.reflect.Method.invoke(Native Method)
12-24 10:59:56.285 E/ACRA    ( 1612): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
12-24 10:59:56.285 E/ACRA    ( 1612): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003)
12-24 10:59:56.285 E/ACRA    ( 1612): 	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@ccc7bba, Dispatchers.Default]
12-24 10:59:56.287 D/ACRA    ( 1612): Building report
12-24 10:59:56.296 D/ACRA    ( 1612): Starting collectors with priority FIRST
12-24 10:59:56.297 D/ACRA    ( 1612): Calling collector org.acra.collector.StacktraceCollector
12-24 10:59:56.298 D/ACRA    ( 1612): Calling collector org.acra.collector.DropBoxCollector
12-24 10:59:56.298 D/ACRA    ( 1612): Collector org.acra.collector.DropBoxCollector completed
12-24 10:59:56.300 D/ACRA    ( 1612): Calling collector org.acra.collector.LogCatCollector
12-24 10:59:56.300 D/ACRA    ( 1612): Collector org.acra.collector.StacktraceCollector completed
12-24 10:59:56.302 D/DefaultMediaStreamState( 1612): Current stream changed to PlayableMediaStream(identifier=9a1abccb69334f6f85d3546e3515ccb8, conversionMethod=None, container=MediaStreamContainer(format=flac), tracks=[MediaStreamAudioTrack(codec=flac, bitrate=1789151, channels=2, sampleRate=48000)], queueEntry=org.jellyfin.playback.jellyfin.queue.item.BaseItemDtoUserQueueEntry@ee405b9, url=http://192.168.0.100:8096/Audio/63b6ee2c-f493-65ec-baff-271fcd87232e/stream?static=true&playSessionId=9a1abccb69334f6f85d3546e3515ccb8&mediaSourceId=63b6ee2cf49365ecbaff271fcd87232e&streamOptions=%7B%7D)
12-24 10:59:56.307 D/MediaCodec( 1612): keep callback message for reclaim
12-24 10:59:56.307 D/AudioNowPlayingFragment( 1612): **** Got playstate change: PAUSED
12-24 10:59:56.307 I/CCodecConfig( 1612): query failed after returning 8 values (BAD_INDEX)
12-24 10:59:56.308 W/Codec2Client( 1612): query -- param skipped: index = 1342179345.
12-24 10:59:56.308 W/Codec2Client( 1612): query -- param skipped: index = 2415921170.
12-24 10:59:56.308 D/AudioNowPlayingFragment( 1612): Updating buttons
12-24 10:59:56.324 D/ACRA    ( 1612): Retrieving logcat output (buffer:default)...
12-24 10:59:56.358 I/org.jellyfin.sdk.api.client.KtorClient( 1612): POST http://192.168.0.100:8096/Sessions/Playing
12-24 10:59:56.375 I/org.jellyfin.sdk.api.client.KtorClient( 1612): POST http://192.168.0.100:8096/Sessions/Playing/Progress

```  

### App information

***App version***: `0.16.2` (`160299`)  
***Package name***: `org.jellyfin.androidtv`  
***Build***: 
```json
{"BOARD":"sabrina","BOOTLOADER":"01.01.231012.083321","BRAND":"google","CPU_ABI":"armeabi-v7a","CPU_ABI2":"armeabi","DEVICE":"sabrina","DISPLAY":"STTE.230915.005.S1.10942527","FINGERPRINT":"google\/sabrina_prod_stable\/sabrina:12\/STTE.230915.005.S1\/10942527:user\/release-keys","HARDWARE":"amlogic","HOST":"abfarm-release-2004-0153","ID":"STTE.230915.005.S1","IS_DEBUGGABLE":false,"IS_EMULATOR":false,"MANUFACTURER":"Google","MODEL":"Chromecast","ODM_SKU":"US","PERMISSIONS_REVIEW_REQUIRED":true,"PRODUCT":"sabrina_prod_stable","RADIO":"unknown","SKU":"US","SOC_MANUFACTURER":"Amlogic","SOC_MODEL":"AMLS905D3","SUPPORTED_32_BIT_ABIS":["armeabi-v7a","armeabi"],"SUPPORTED_64_BIT_ABIS":[],"SUPPORTED_ABIS":["armeabi-v7a","armeabi"],"TAGS":"release-keys","TIME":1697143988000,"TYPE":"user","UNKNOWN":"unknown","USER":"android-build","VERSION":{"ACTIVE_CODENAMES":[],"BASE_OS":"","CODENAME":"REL","INCREMENTAL":"10942527","MEDIA_PERFORMANCE_CLASS":0,"PREVIEW_SDK_FINGERPRINT":"REL","PREVIEW_SDK_INT":0,"RELEASE":"12","RELEASE_OR_CODENAME":"12","SDK":"31","SDK_INT":31,"SECURITY_PATCH":"2023-10-01"}}
```  
***Build config***: 
```json
{"APPLICATION_ID":"org.jellyfin.androidtv","BUILD_TYPE":"release","DEBUG":false,"DEVELOPMENT":false,"VERSION_CODE":160299,"VERSION_NAME":"0.16.2"}
```  

### Device information

Chromecast with Google TV 4K

***Android version***: `12`  
***Device brand***: `google`  
***Device product***: `sabrina_prod_stable`  
***Device model***: `Chromecast`  

### Crash information

***Start time***: `2023-12-24T10:47:38.832+07:00`  
***Crash time***: `2023-12-24T10:59:56.765+07:00`  

### Where did you install the app from?

Google Play, enrolled to Beta program

### Jellyfin server version

10.8.13

**Comment by nielsvanvelzen** [comment]
Thanks for the report. I've identified multiple issues with the shuffle code now and they should all be fixed in 0.16.3!

**Comment by BobbyWibowo** [comment]
Awesome, thanks for the quick fix, will be looking forward for the next update!
